ee/tabby-webserver/src/service/background_job/web_crawler.rs

```diff
@@ -20,8 +24,17 @@ impl WebCrawlerJob {
         Self { url }
     }
 
-    pub async fn run(self, embedding: Arc<dyn Embedding>) {
-        tabby_scheduler::crawl_index_docs(&[self.url], embedding).await;
+    pub async fn run(self, job_logger: JobLogger, embedding: Arc<dyn Embedding>) -> Result<()> {
+        tabby_scheduler::crawl_index_docs(
+            &[self.url],
+            move |url| {
+                let job_logger = job_logger.clone();
+                async move { cprintln!(job_logger, "{url}") }
```

Comment:
Please write more informative logs since it's user facing, maybe something like: "Fetching {url}"